components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -980,12 +980,16 @@ private void appendFilterQuery(List<ExpressionNode> expressionNodes, FilterQuery
      * @param idpId        IDP Id
      * @return Identity provider properties
      */
-    private List<IdentityProviderProperty> getIdentityPropertiesByIdpId(Connection dbConnection, int idpId)
+    private List<IdentityProviderProperty> getIdentityPropertiesByIdpId(Connection dbConnection,
+                                                                        int idpId, int tenantId)
             throws SQLException {
 
+        final String DEPRECATED_RECOVERY_NOTIFICATION_PASSWORD_CONFIG = "Recovery.Notification.Password.Enable";
```

Comment:
Lets introduce a constant for this

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/5601/commits/dc5434b3928be58732d3010cbe33a64acb9825de